app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/room/LiveDataDetailActivity.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package android.support.v17.leanback.supportleanbackshowcase.app.room;
+
+import android.arch.lifecycle.LifecycleRegistry;
+import android.arch.lifecycle.LifecycleRegistryOwner;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v17.leanback.supportleanbackshowcase.R;
+import android.support.v4.app.FragmentActivity;
+
+
+public class LiveDataDetailActivity extends FragmentActivity implements LifecycleRegistryOwner{
```

Comment:
any reason not subclass LifeCycleActivity,  or "MyFragment extends BrowseSupportFragment implements LifecycleRigstryOwner"?

Reply:
Sure, will use LifeCycleActivity

Reply:
Done